src/commands/misc/covid.js

```diff
@@ -0,0 +1,14 @@
+const { SubcommandListCommand, Constants } = require('../../')
+
+module.exports = class Covid extends SubcommandListCommand {
+  constructor (client) {
+    super({
+      name: 'covid',
+      aliases: ['covid19', 'coronavirus'],
+      authorString: 'commands:covid.covid',
+      authorImage: 'https://i.imgur.com/Rnobe3k.png',
+      authorURL: 'https://covid19.who.int/',
+      embedColor: Constants.NPM_COLOR
```

Comment:
Add a separate color for this command instead of using NPM's

Reply:
theres no problem using the npm actually, i wanted to use a red color.